types/Actions.d.ts

```diff
@@ -167,6 +168,11 @@ declare module 'grafx-studio-actions' {
         selected?: string;
     }
 
+    export interface BooleanVariable extends BaseVariable {
+        type: VariableType.boolean;
+        value: boolean;
+    }
+
     interface GroupVariable extends BaseVariable {
         type: VariableType.group;
```

Comment:
Sorry, just noticed, but why there is no `value` for this variable type? If it is present for all other types.

Reply:
Indeed, group is just a holder variable to group children. It has no value, children refer to the group by its id.